src/main/java/com/denizenscript/depenizen/bukkit/properties/worldguard/WorldGuardPlayerExtensions.java

```diff
@@ -0,0 +1,144 @@
+                    return null;
+                }
+                WorldGuardPlugin worldGuard = (WorldGuardPlugin) WorldGuardBridge.instance.plugin;
+                return new ElementTag(WorldGuard.getInstance().getPlatform().getRegionContainer().createQuery().testBuild(BukkitAdapter.adapt(loc), worldGuard.wrapPlayer(player.getPlayerEntity())));
+            }
+
+            // <--[tag]
+            // @attribute <PlayerTag.worldguard.test_flag[<name>]>
+            // @returns ObjectTag
+            // @plugin Depenizen, WorldGuard
+            // @deprecated Use 'PlayerTag.worldguard_flag'
+            // @description
+            // Deprecated in favor of <@link tag PlayerTag.worldguard_flag>
+            // -->
+            if (attribute.startsWith("test_flag", 2) && attribute.hasContext(2)) {
+                worldguardTestFlag.warn(attribute.context);
+                ElementTag name = attribute.contextAsType(2, ElementTag.class);
+                attribute.fulfill(1);
+                Flag<?> flag = Flags.fuzzyMatchFlag(WorldGuard.getInstance().getFlagRegistry(), name.asString());
+                if (flag == null) {
+                    Debug.echoError("The tag PlayerTag.worldguard.test_flag[...] has an invalid value: " + name.asString());
+                    return null;
+                }
+                LocationTag loc = player.getLocation();
+
+                // <--[tag]
+                // @attribute <PlayerTag.worldguard.test_flag[<name>].at[<location>]>
+                // @returns ObjectTag
+                // @plugin Depenizen, WorldGuard
+                // @deprecated Use 'PlayerTag.worldguard_flag'
+                // @description
+                // Deprecated in favor of <@link tag PlayerTag.worldguard_flag>
+                // -->
+                if (attribute.startsWith("at", 2) && attribute.hasContext(2)) {
+                    loc = attribute.contextAsType(2, LocationTag.class);
+                    attribute.fulfill(1);
+                    if (loc == null) {
+                        return null;
+                    }
+                }
+                WorldGuardPlugin worldGuard = (WorldGuardPlugin) WorldGuardBridge.instance.plugin;
+                RegionQuery query = WorldGuard.getInstance().getPlatform().getRegionContainer().createQuery();
+                if (flag instanceof StateFlag stateFlag) {
+                    return new ElementTag(query.testState(BukkitAdapter.adapt(loc), worldGuard.wrapPlayer(player.getPlayerEntity()), stateFlag));
+                }
+                else {
+                    return CoreUtilities.objectToTagForm(query.queryValue(BukkitAdapter.adapt(loc), worldGuard.wrapPlayer(player.getPlayerEntity()), flag), attribute.context).asElement();
+                }
+            }
+
+            return null;
+        };
+
+        PlayerTag.tagProcessor.registerTag(ObjectTag.class, "worldguard", runnable);
+        PlayerTag.tagProcessor.registerTag(ObjectTag.class, "wg", runnable);
```

Comment:
This `wg` part can be dropped - there were scraps of code about it but it wasn't functionally implemented (not listed in `handledTags`)